commands/base_test.go

```diff
@@ -110,3 +113,63 @@ func TestReadFile(t *testing.T) {
 		require.Empty(t, certificate)
 	})
 }
+
+type cmdWithDefaultsApply struct {
+	applyDefaultCalled bool
+	user               string
+	password           string
+}
+
+func (c *cmdWithDefaultsApply) ApplyDefaults(cfg *ini.File) {
+	c.user = cfg.Section("client").Key("user").String()
+	c.password = cfg.Section("client").Key("password").String()
+	c.applyDefaultCalled = true
+}
+
+func TestConfigureDefaults(t *testing.T) {
+	t.Run("ApplyDefaults is called if command supports it", func(t *testing.T) {
+		file, e := os.Open("../testdata/.my.cnf")
```

Comment:
@BupycHuk you were asking for it. I use real file in cmd test, but tmp files for  `commands/management/add_mysql.go`